assets/js/main.js

```
@@ -1,778 +1,795 @@
+            const target = parseInt(counter.textContent.replace(/\D/g, ''));
+            const duration = 2000;
+            const step = target / (duration / 16);
+            let current = 0;
+
+            const updateCounter = () => {
+                current += step;
+                if (current < target) {
+                    counter.textContent = Math.floor(current) + (counter.textContent.includes('+') ? '+' : '');
+                    requestAnimationFrame(updateCounter);
+                } else {
+                    counter.textContent = counter.textContent; // Reset to original
```

Biome (2.1.2)

[error] textContent is assigned to itself.

This is where is assigned.

Self assignments have no effect and can be removed.

(lint/correctness/noSelfAssign)